setupbase.py

```diff
@@ -153,7 +153,6 @@ def find_package_data():
         pjoin(components, "underscore", "underscore-min.js"),
         pjoin(components, "moment", "moment.js"),
         pjoin(components, "moment", "min", "moment.min.js"),
-        pjoin(components, "term.js", "src", "term.js"),
```

Comment:
These are explicit checks for bower modules, there is no need IMO to check for node_modules.